src/main/java/se/sundsvall/parkingpermit/service/mapper/RpaMapper.java

```diff
@@ -0,0 +1,17 @@
+package se.sundsvall.parkingpermit.service.mapper;
+
+import generated.se.sundsvall.rpa.QueueItemDataDto;
+import generated.se.sundsvall.rpa.QueuesAddQueueItemParameters;
+
+import static java.util.Objects.isNull;
+
+public class RpaMapper {
+
+	private RpaMapper() {}
+
+	public static QueuesAddQueueItemParameters toQueuesAddQueueItemParameters(String queueName, Long caseId) {
+		return new QueuesAddQueueItemParameters().itemData(new QueueItemDataDto()
+			.name(queueName)
+			.reference(isNull(caseId) ? null : String.valueOf(caseId)));
```

Comment:
Use ofNullable instead? I.e. change `caseId) ? null : String.valueOf(caseId)` to `ofNullable(caseId).map(String::valueOf).orElse(null)`